### DISCUSSION TITLE
Vencord Compatibility

### DISCUSSION THREAD

**Discussion by vinylll04** [maltejur/discord-screenaudio#183] [General]
I tried installing vencord to the flatpak file location, it ran and said it was doing stuff but it never updated to the flatpak of discord-screenaudio. Is this because its just not compatible yet or what are the issues that need to be fixed in order for vencord to become compatible?

**Comment by maltejur** [comment]
This program is more like a web browser which wraps the discord website, so the desktop installer of vencord won't work. I did work a bit on integrating the web version in https://github.com/maltejur/discord-screenaudio/issues/59, but I currently don't have the time to finish or maintain that.
